### ISSUE TITLE
#779 regulator and CT/PT ratios,


### ISSUE THREAD

**Issue by nikhilgupta10** [gridlab-d/gridlab-d#779]
CT and PT ratios only work with line drop compensation. it is common for commercial tools to always specify all control modes in \p.u.\, i.e., 120 V bandcenter +/- 1 V bandwidth (rather than 7200 +/- 120).  

,

**Comment by ftuffner** [comment]
This recently came up in a forum post, so it may be time to actually fix this one (should be pretty easy).

**Comment by temcdrm** [comment]
Yes, except that we actually want to say that **band_center** and **band_width** are in secondary volts, usually 120 nominal.  Not primary volts, not per-unit, not percent.

Also, can we please change **power_transducer_ratio** to **voltage_transducer_ratio**?  Strictly speaking, VT is now the correct acronym in place of PT, and even when PT was correct, the P stood for **potential**, not power.  Alternatively, **power_transducer_ratio** could still be accepted, but deprecated in favor of **voltage_transducer_ratio**.

Lastly, the help for both of the transducer ratio attributes should say **turns ratio** instead of rating.
